media-art-background.js

```diff
@@ -17,23 +18,37 @@ function setBackground(root, lovelace, app, originalStyle) {
     const backgroundUrl = entityInfo.attributes[entityImageAttribute];
     if (!backgroundUrl) continue;
 
-    app.style.backgroundImage = `url('${backgroundUrl}')`
-    app.style.backgroundRepeat = 'no-repeat';
-    app.style.backgroundPosition = 'center';
-    app.style.backgroundSize = 'cover';
-    app.style.backgroundAttachment = 'fixed';
+    bground_elem.style.backgroundImage = `url('${backgroundUrl}')`
+    bground_elem.style.backgroundRepeat = 'no-repeat';
+    bground_elem.style.backgroundPosition = 'center';
+    bground_elem.style.backgroundSize = 'cover';
+    bground_elem.style.backgroundAttachment = 'fixed';
+    bground_elem.style.filter = `blur(${filterBlur})`;
 
     return;
   }
 
-  app.style = originalStyle;
+  bground_elem.style = originalStyle;
 };
 
 let root = document.querySelector("home-assistant");
 root = root.shadowRoot.querySelector("home-assistant-main").shadowRoot.querySelector("app-drawer-layout partial-panel-resolver ha-panel-lovelace").shadowRoot.querySelector("hui-root");
 const lovelace = root.lovelace;
 const app = root.shadowRoot.querySelector("ha-app-layout");
-const originalStyle = app.style;
 
-setInterval(function() { setBackground(root, lovelace, app, originalStyle) }, 5000);
-setBackground(root, lovelace, app, originalStyle);
+const bground = document.createElement("div"); // create empty container for background 
+
+bground.style.position = "absolute"; // fill entire window
+bground.style.top = 0;
+bground.style.left = 0;
+bground.style.width = "100%";
+bground.style.height = "100%";
+
+bground.style.zIndex = -1; // below view elements
+
+app.appendChild(bground);
+
+const originalStyle = bground.style;
```

Comment:
Hmm, not sure if this style resetting is still the best way because of adding a div now. But should still work fine so sure :)